plasma/client/child_chain_service.py

```diff
@@ -22,8 +22,7 @@ def send_request(self, method, args):
             "jsonrpc": "2.0",
             "id": 0,
         }
-        response = requests.post(self.url, data=json.dumps(payload),
-                                 headers=headers).json()
+        response = requests.post(self.url, data=payload).json()
```

Comment:
Should be `json=payload` if we want to send as JSON w/o `headers = {'content-type': 'application/json'}`. Might as well remove `headers = {'content-type': 'application/json'}` if we're going this route.